sunpy/map/sources/sdo.py

```diff
@@ -47,6 +49,43 @@ def __init__(self, data, header, **kwargs):
         self.plot_settings['cmap'] = plt.get_cmap(self._get_cmap_name())
         self.plot_settings['norm'] = ImageNormalize(stretch=source_stretch(self.meta, AsinhStretch(0.01)))
 
+    @property
+    def observer_coordinate(self):
+        """
+        The Heliographic Stonyhurst Coordinate of the observer.
+        The HAE coordinates in the header are used instead of the HGS coordinates in the header.
+        """
+        vector = CartesianRepresentation(self.meta['haex_obs'],
+                                         self.meta['haey_obs'],
+                                         self.meta['haez_obs'])
+        coord = SkyCoord(vector * u.m, frame=HeliocentricTrueEcliptic, obstime=self.date)
+        return coord.heliographic_stonyhurst
+
+    @property
+    def heliographic_latitude(self):
+        """Heliographic latitude (calculated from HAE coordinates)."""
```

Comment:
I think it could be better if we put this comment in the `observer_coordinate` docstring and not in all of these?